src/NServiceBus.Metrics/NServiceBus.Metrics.csproj

```diff
@@ -16,7 +16,7 @@
   <ItemGroup>
     <PackageReference Include="ILRepack" Version="2.*" PrivateAssets="All" />
     <PackageReference Include="Metrics.NET" Version="0.4.8" PrivateAssets="All" />
-    <PackageReference Include="NServiceBus" Version="[6.*, 7.0.0)" />
+    <PackageReference Include="NServiceBus" Version="[6.2.0, 7.0.0)" />
```

Comment:
My assumption is that we want to work for any minor within major 6. Shouldn't it be `6.0.0`?

Reply:
We need at least `6.2` as it introduced the `ReceivePipelineCompleted` event to public API. Metrics plugin uses that for probes implementations. See https://github.com/Particular/NServiceBus/pull/4509 for details on core changes.